migrations/1597882126530_create-collections.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+import { Collection } from '../src/Collection'
+
+const tableName = Collection.tableName
+
+export const up = (pgm: MigrationBuilder) => {
+  pgm.createTable(
+    tableName,
+    {
+      id: { type: 'UUID', primaryKey: true, unique: true, notNull: true },
+      name: { type: 'TEXT', notNull: true },
+      eth_address: { type: 'TEXT', notNull: true },
+      hash: { type: 'UUID' },
```

Comment:
I'd suggest using  name `salt` and type `TEXT` which is better to store a 32 bytes varchar. `hash` is more generic. In some languages is a reserved word.

Reply:
Gotcha!
The UUID was a slipup btw